dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -296,6 +305,31 @@ if [ -n "${WLP_SKIP_BOOTSTRAP_AGENT}" ]; then
   JAVA_AGENT_QUOTED=
 fi
 
+##
+## Determine if CRIU supports unprivileged mode.
+## Determine which CRIU mode to use, based on current euid, whether or not criu supports unprivileged mode, and the value of the CRIU_UNPRIVILEGED env var.
+## Set DO_CRIU_UNPRIVILEGED based on the above.
+##
+checkCriuUnprivileged()
+{
+  # Determine if CRIU supports unprivileged mode
+  criu --help | grep -q -e '--unprivileged' && CRIU_SUPPORTS_UNPRIVILEGED=true || CRIU_SUPPORTS_UNPRIVILEGED=false
+
+  # Determine which CRIU mode to use, based on current euid and values of CRIU_SUPPORTS_UNPRIVILEGED and CRIU_UNPRIVILEGED
+  DO_CRIU_UNPRIVILEGED=false
+  if [ "$(id -u)" != 0 ]; then
+    # Not root, unprivileged by default if CRIU supports it
+    if [ $CRIU_SUPPORTS_UNPRIVILEGED = true -a "${CRIU_UNPRIVILEGED}" != "false" -a "${CRIU_UNPRIVILEGED}" != "1" -a "${CRIU_UNPRIVILEGED}" != "FALSE" -a "${CRIU_UNPRIVILEGED}" != "False" ]; then
```

Comment:
Just noticed the `"${CRIU_UNPRIVILEGED}" != "1"` part should be `"${CRIU_UNPRIVILEGED}" != "0"`.

Reply:
Fixed.

Reply:
My latest commit removes the false check altogether.  If not set to one of the "true" values then it just sets `DO_CRIU_UNPRIVILEGED` to false.

I also moved the check for the `CRIU_UNPRIVILEGED` variable to be first and if set then it never does a check for support in a `criu` help check.  @ymanton let me know if you see an issue with that.  I figured if someone sets the variable then just take what they specified and there is no need to run a `criu` to see if it is supported.

Reply:
Looks good, it reads clearly to me. I updated the comment as a suggestion.